models/User.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+//TODO's
+//create key for user avatar
+
+const UserSchema = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+    lowercase: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  primaryLanguage: {
+    type: String,
+    required: true,
+  },
+  contacts: [],
```

Comment:
I think we do need to define something here. We can probably use the way shared in this article: https://www.initialapps.com/mongoose-why-you-may-be-having-issues-populating-across-multiple-levels/

Reply:
^ Approved to merge after you fix this one thing

Reply:
fixing now.  Thank you